site/pages/view-all-templates.page.tsx

```diff
@@ -1,53 +1,39 @@
+import { Progress } from "@create-disruptions-data/shared-ts/enums";
 import { NextPageContext } from "next";
-import Link from "next/link";
 import { ReactElement } from "react";
 import { randomUUID } from "crypto";
 import { BaseLayout } from "../components/layout/Layout";
+import ViewAllContents, { ViewAllContentProps } from "../components/ViewAllContents";
 import { getSessionWithOrgDetail } from "../utils/apiUtils/auth";
 
 const title = "View All Templates";
 const description = "View All Templates page for the Create Transport Disruptions Service";
 
-export interface ViewAllTemplatesProps {
-    newDisruptionId: string;
-    csrfToken?: string;
-}
-
-const ViewAllTemplates = ({ newDisruptionId }: ViewAllTemplatesProps): ReactElement => {
+const ViewAllTemplates = ({
+    newContentId,
+    adminAreaCodes,
+    filterStatus,
+    enableLoadingSpinnerOnPageLoad = true,
+}: ViewAllContentProps): ReactElement => {
     return (
         <BaseLayout title={title} description={description}>
-            <h1 className="govuk-heading-xl">View all templates</h1>
```

Comment:
the header says view all disruptions i think it should be view all templates

Reply:
done